copy-reports/copy-reports.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+set -euxo pipefail
+
+# Prompt user for namespace and validate
+read -p "Enter the namespace for your kubecost deployment: " NAMESPACE
+
+if [ -z "$NAMESPACE" ]; then
+    echo "Namespace cannot be empty. Exiting."
+    exit 1
+fi
+
+# Check if the provided namespace exists
+if ! kubectl get namespace "$NAMESPACE" &>/dev/null; then
+    echo "Namespace '$NAMESPACE' does not exist. Exiting."
+    exit 1
+fi
+
+# Check if aggregator is deployed as a statefulset
+if helm get values kubecost -n $NAMESPACE | grep -q "deployMethod: statefulset"; then
```

Comment:
I like the approach - however, you would then need the user to provide the name of their release as a parameter as well. Here's a bit of context that might help us simplify this: 

> In a kubecost deployment, there is only one time that a statefulset with the label `app: aggregator` is deployed - when kubecost is running in aggregator statefulset mode

So, you could get sts with the label app: aggregator in the kubecost namespace, which we already asked the user for. If no results returned, then you fail the guard. for example: 
```
$ kubectl -n alex get sts -l app=aggregator
NAME                 READY   AGE
kc-alex-aggregator   1/1     4h46m

 kubectl -n kube-system  get sts -l app=aggregator
No resources found in kube-system namespace.
```